client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -80,12 +86,40 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
             }
         }
 
+        MapType mapType = mapper.getTypeFactory().constructMapType(HashMap.class, String.class,
+                SystemResourceLimits.class);
+        Map<String, SystemResourceLimits> systemResourceLimits = new HashMap<>();
+        if (systemLimits != null && !systemLimits.isEmpty()) {
```

Comment:
nit - `Utils.isEmpty`?

Reply:
The utils library is not brought in the cli. I don't feel strongly it's necessary.